src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.report;
+
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");
+        for (Map.Entry<String, Integer> entry : Storage.getFruitStorage().entrySet()) {
```

Comment:
Follow the encapsulation principle. Consider making `Storage.getFruitStorage()` return a copy of the storage map or an unmodifiable view to prevent exposing the internal structure.